src/test/ui/did_you_mean/use_instead_of_import.rs

```diff
@@ -0,0 +1,10 @@
+import std::{
```

Comment:
Probably doesn't matter but it might be good to include the other style of path as well for completeness (`import std::sync::Arc;`)